storage/remote/write_handler_test.go

```diff
@@ -204,6 +204,7 @@ func TestCommitErr(t *testing.T) {
 }
 
 func BenchmarkRemoteWriteOOOSamples(b *testing.B) {
```

Comment:
@jesusvazquez if you have some time we should rework this benchmark, I can open an issue for it